### PR TITLE
Prepare download links for v0.4.0

### PR DESCRIPTION
**Pull request by staticfloat**

### CODE REVIEW

**Comment by tkelman** [comment]
Yay. Did you keep https://github.com/JuliaLang/julia/issues/13456 in mind when making the source-dist tarballs?

**Comment by staticfloat** [comment]
Yep.  :)  They should be named `julia-0.4.0.tar.gz` and `julia-0.4.0-full.tar.gz`, and should extract to `julia-0.4.0/`.

File: downloads/index.md [hunk 1]
```diff
@@ -23,43 +23,44 @@ packages are pre-installed.
...
+    <td colspan="3"> <a href="https://julialang.s3.amazonaws.com/bin/linux/x64/0.4/julia-0.4.0-linux-x86_64.tar.gz">64-bit</a> (<a href="https://julialang.s3.amazonaws.com/bin/linux/x64/0.4/julia-0.4.0-linux-x86_64.tar.gz.asc">GPG</a>)</td>
 </tr>
 <tr>
     <th> Source </th>
-    <td> <a href="https://github.com/JuliaLang/julia/releases/download/v0.3.11/julia-0.3.11_483dbf5279.tar.gz">Tarball</a> (<a href="https://github.com/JuliaLang/julia/releases/download/v0.3.11/julia-0.3.11_483dbf5279.tar.gz.asc">GPG</a>) </td>
-    <td> <a href="https://github.com/JuliaLang/julia/tree/release-0.3">GitHub</a> </td>
+    <td colspan="2"> <a href="https://github.com/JuliaLang/julia/releases/download/v0.4.0/julia-0.4.0.tar.gz">Tarball</a> (<a href="https://github.com/JuliaLang/julia/releases/download/v0.4.0/julia-0.4.0.tar.gz.asc">GPG</a>) </td>
+    <td colspan="2"> <a href="https://github.com/JuliaLang/julia/releases/download/v0.4.0/julia-0.4.0-full.tar.gz">Tarball</a> (<a href="https://github.com/JuliaLang/julia/releases/download/v0.4.0/julia-0.4.0-full.tar.gz.asc">GPG</a>) </td>
```
Comment (tkelman) [inline downloads/index.md]: link should say "Full tarball"

**Comment by tkelman** [comment]
> and should extract to `julia-0.4.0/`

poifect. Maybe we should figure out a way to patch the makefiles to do this automatically before the next one if you don't want to remember to do it by hand each time.

**Comment by staticfloat** [comment]
If you didn't take the words out of my mouth, you at least stole them from my fingers.

**Pull request merged**